### ISSUE TITLE
Toggle For Abilities?

### ISSUE THREAD

**Issue by Kyanlte** [CyclopsMC/EverlastingAbilities#39]
I noticed that for abilities like levitation it affects everything constantly, and always and deactivating the power with my totem is a bit of a hassle, would it be possible to add a toggle for powers?

**Comment by rubensworks** [comment]
This would definitely be useful. But it should be well thought out, because abilities should be disableable separately.

**Comment by Kyanlte** [comment]
Good idea! It would be bad if you were flying really high up and accidentally turned it off.

**Comment by scottwilson0308** [comment]
I was just about to make a new issue suggesting this, as I've been thinking of this for quite a while, thanks to Magnetize.. I keep my ability bottle in my inventory just so I can disable it (frequently). 

On a side note, I finally just got Flight on my Skyblock map.. EA FTW!

**Comment by scottwilson0308** [comment]
Just discovered by chance that Magnetize disables upon sneaking.. :)

**Comment by markygnlg** [comment]
Hmmm, no new info but a reminder to us about crouch/sneaking + 'Magnetize' or 'Power Stare' is something to test ...
